security/compliance_operator/compliance-operator-supported-profiles.adoc

```diff
@@ -6,6 +6,11 @@ include::_attributes/common-attributes.adoc[]
 
 There are several profiles available as part of the Compliance Operator (CO) installation.
 
+[IMPORTANT]
+====
+The Compliance Operator might report incorrect results on managed platforms, such as OpenShift Dedicated, Red Hat OpenShift Service on AWS, and Azure Red Hat OpenShift.
```

Comment:
We are tracking work to enable the Compliance Operator on managed offerings, like ROSA. Do we want to include a link to that? Or do we just want to say that this will be addressed with future versions of the operator (e.g., 0.1.57+)?

Reply:
Ah - here's how this could play out. 

We could say "For more information refer to the [KCS article]."

The article can contain the link to a Jira ticket or anything else. But I don't want to say "we are working on it" in public documentation. The KCS would at least be a soft (free?) pay wall. 

What do you think?

Reply:
I think that approach sounds good.

Reply:
Cool - I DM'ed you a rough outline of a KCS on slack. When we have that settled and published, I can then add the URL to this pull request.